src/components/ProductsCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useNavigate } from 'react-router-dom';
+
+const ProductsCard = ({ products }) => {
+    const navigate = useNavigate();
+    const pageId = products.name.stringValue.toLowerCase();
```

Comment:
Are we lowercasing this b/c the react router is case sensitive when navigating to different urls?

Reply:
yes sir